ls1intum/Artemis
# Fix count programming exercises

### Checklist

### Motivation and Context


There is a problem with counts within the exercise overview dashboard for normal programming exercises (not within exams). 
This PR fixes the problem, now all counts within normal exercises should work correctly. 
We also removed the 'information' Button within the header of the assessment dashboard and made the header itself a bit smaller.

### Description


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> course -> assessment dashboard
3. Check if all progress bars display correct counts 

### Screenshots


Smaller header:
<img width="1394" alt="Bildschirmfoto 2021-02-23 um 09 58 19" src="https://user-images.githubusercontent.com/42205689/108821554-10bd2780-75be-11eb-8ced-e3cc18f7b958.png">
